platform/kube/inject/inject.go

```diff
@@ -40,7 +40,9 @@ import (
 // Defaults values for injecting istio proxy into kubernetes
 // resources.
 const (
+	DefaultHubEnvVar       = "MANAGER_HUB"
```

Comment:
I put the new constants with the old constants.

I can't reason about the best structure for these constants because we plan to support multiple platforms soon.  The other platforms will have different default constants and probably entirely different flags in some cases.  Keeping the Kubernetes specific constants in platform/kube reinforces that.